File: 프로그래머스/0/181900. 글자 지우기/글자 지우기.py
def solution(my_string, indices):
    # method1
    # idx_set=set([i for i in range(len(my_string))])
    # r=''.join(my_string[i] for i in sorted(idx_set.difference(set(indices))))
    # method2
    r=''.join(my_string[i] for i in range(len(my_string)) if i not in indices)
    return r

File: 프로그래머스/0/181900. 글자 지우기/글자 지우기.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

string solution(string my_string, vector<int> indices) {
    sort(indices.begin(),indices.end(),less<int>());
    bool flag;
    string r="";
    // for(int i:indices) {
    //     my_string.erase(i,1);
    // }
    
    for(int i=0;i<my_string.size();i++) {
        flag=true;
        for(int j=0;j<indices.size();j++)
            if(i==indices[j]) {
                flag=false;
                break;
            }
        if(flag)
            r+=my_string[i];    
    }
    return r;
}